In [45]:
import iris
import glob
import warnings
warnings.filterwarnings('ignore')

In [49]:
vol_files = glob.glob('/g/data/ua6/DRSv2/CMIP5/*/historical/fx/ocean/r0i0p0/volcello/latest/*.nc')
for vol_file in vol_files:        
    vol_cube = iris.load_cube(vol_file, 'ocean_volume')
    print(vol_file)
    
    coord_names = [coord.name() for coord in vol_cube.dim_coords]
    print(coord_names)
    print(vol_cube.data.shape)
    print(vol_cube.data.sum())

/g/data/ua6/DRSv2/CMIP5/ACCESS1-3/historical/fx/ocean/r0i0p0/volcello/latest/volcello_fx_ACCESS1-3_historical_r0i0p0.nc
['depth', 'cell index along second dimension', 'cell index along first dimension']
(50, 300, 360)
1.29137e+18
/g/data/ua6/DRSv2/CMIP5/CSIRO-Mk3-6-0/historical/fx/ocean/r0i0p0/volcello/latest/volcello_fx_CSIRO-Mk3-6-0_historical_r0i0p0.nc
['depth', 'latitude', 'longitude']
(31, 189, 192)
1.33911e+18
/g/data/ua6/DRSv2/CMIP5/IPSL-CM5A-LR/historical/fx/ocean/r0i0p0/volcello/latest/volcello_fx_IPSL-CM5A-LR_historical_r0i0p0.nc
['depth', 'cell index along second dimension', 'cell index along first dimension']
(31, 149, 182)
2.62012e+18
/g/data/ua6/DRSv2/CMIP5/MIROC5/historical/fx/ocean/r0i0p0/volcello/latest/volcello_fx_MIROC5_historical_r0i0p0.nc
['depth', 'grid_latitude', 'grid_longitude']
(50, 224, 256)
1.36273e+18
/g/data/ua6/DRSv2/CMIP5/MIROC-ESM/historical/fx/ocean/r0i0p0/volcello/latest/volcello_fx_MIROC-ESM_historical_r0i0p0.nc
['ocean sigma over z coordinate', 'lat

For most models, the volume of the global ocean is approximately $1.3 \times 10^{18} m^{-3}$. It is an order of magnitude less for the EC-EARTH and CCSM4 models. Why?